Contributing.md

```diff
@@ -59,7 +59,7 @@ example, the `issue-1111.rs` test file is configured by the file
 ## Debugging
 
 Some `rewrite_*` methods use the `debug!` macro for printing useful information.
-These messages can be printed by using the environment variable `RUSTFMT_LOG=rustfmt=DEBUG`.
+These messages can be printed by using the environment variable `RUSTFMT_LOG="rustfmt=DEBUG"`.
```

Comment:
Right now I'm wondering if it would be better to mention `RUSTFMT_LOG=rustfmt=DEBUG` on linux and add a new sentence to make a distinction for `RUSTFMT_LOG="rustfmt=DEBUG"` when using powershell on windows.

Reply:
The value is a String, it needs the quote. "RUSTFMT_LOG=rustfmt=DEBUG" is just lazy and not formal.

Reply:
@rzhb first I want to highlight that I appreciate you helping to improve the docs, but I don't find the criticism to be constructive.

I wouldn't characterize this as lazy since the quotes are unnecessary when working on linux / mac. Given that the current docs are valid for some systems, I suggest making an amendment to the docs that informs powershell users that they need to wrap `rustfmt=DEBUG` in quotes. Are the quotes also necessary when running in the windows command prompt?

Reply:
On Linux, without quotes works only because there has no whitespace or special characters.
[It's better to always quote string for safety.](https://stackoverflow.com/questions/10067266/when-to-wrap-quotes-around-a-shell-variable)

powershell     
`$env:RUSTFMT_LOG="rustfmt=DEBUG"` work.
`$RUSTFMT_LOG="rustfmt=DEBUG"` didn't.

cmd
`set RUSTFMT_LOG="rustfmt=DEBUG"` didn't work.  `rustfmt` has warning `warning: invalid logging spec 'DEBUG"', ignoring it`
`set RUSTFMT_LOG=rustfmt=DEBUG`  work.

Reply:
Thank you for looking into this. From my understanding of your last comment the quotes are only required in powershell, and it seems like the quotes actually break things on cmd. Assuming that's the case I now feel more strongly about leaving the existing language as is and adding a note for powershell users.